Cluster/Cluster Not Reachable  UpgradeServiceNotreachable.md

```diff
@@ -1,37 +1,68 @@
 
-## **Possible Causes**
-- Cluster is down due to seed node Quorum Loss / ring collapsed - too many seed nodes have failed or were brought down at the same time
+## Possible Causes
+
+- Cluster is down due to seed node Quorum Loss / ring collapsed - too many seed nodes failed or were brought down at the same time
 - fabric:/System/UpgradeService (Upgrade service) is down
-- fabric:/System/UpgradeService is unable to reach to the regional Service Fabric Resource Provider (SFRP)
+- fabric:/System/UpgradeService is unable to reach regional Service Fabric Resource Provider (SFRP)
 - TLS 1.0/1.2 was disabled
-- Expired Certificate 
+- Expired Certificate
 
-## **Mitigation Options**
+## Mitigation
 
-- Cluster down / ring collapse
-    - Open support ticket with Microsoft to see if the cluster is recoverable, worst case is a full rebuild of the cluster.
-    - See [Issues caused by Deallocating a VMMS](./Issues%20caused%20by%20Deallocating%20a%20VMSS.md)
-    - See [Common issues caused by AutoScale](./Common%20issues%20customers%20experience%20when%20using%20Auto-scale%20with%20Service%20Fabric%20clusters.md)
+### Cluster down / ring collapse
 
-- fabric:/System/UpgradeService (Upgrade service) is down
-    - May be a transient error, so you can wait and see if it self-corrects
-    - issue with a cluster config upgrade getting stuck while copying the cluster package. This can block all polling by the Upgrade Service.
-        - The workaround is to restart the primary replica of fabric:/System/UpgradeService or you can try to restart the node hosting the primary replica of fabric:/System/UpgradeService
-    - See [Issues caused by Deallocating a VMMS](./Issues%20caused%20by%20Deallocating%20a%20VMSS.md)
-    - See [Common issues caused by AutoScale](./Common%20issues%20customers%20experience%20when%20using%20Auto-scale%20with%20Service%20Fabric%20clusters.md)
+Sometimes the cluster is not recoverable and the worst case is a full rebuild of the cluster.
+
+#### VMs in VMSS associated with SF nodes deallocated
+
+- VMs deallocated / stopped, start the VMs
+- See [Issues caused by Deallocating a VMMS](./Issues%20caused%20by%20Deallocating%20a%20VMSS.md)
+- See [Common issues caused by AutoScale](./Common%20issues%20customers%20experience%20when%20using%20Auto-scale%20with%20Service%20Fabric%20clusters.md)
+
+#### VMs in VMSS associated with SF nodes are in failed state, check status of extensions on these VMs
+
+- Validate that the configuration of failed extensions is correct
+- Service Fabric extension in error, note down the error message if any
+- See [Collecting logs for failed VMs](./Collecting%20logs%20for%20failed%20VMs.md) for obtaining more details about the failure and mitigation
+
+#### VMs healthy
+
+- Service Fabric may be failing to start up on the VM due to various issues
+- RDP to one or more VMs
+- Identify the list of running processes that match Fabric*.exe (in taskmanager)
```

Comment:
Link to https://github.com/Azure/Service-Fabric-Troubleshooting-Guides/blob/master/Cluster/SF%20process%20list%20with%20Security%20Context%20info.md

Reply:
Thanks for the link. We are just interested in the list of processes that match Fabric*.exe. Pointing to the other doc may add unnecessary complexity.